common/eqemu_logsys.cpp

```diff
@@ -532,6 +534,11 @@ void EQEmuLogSys::StartFileLogs(const std::string &log_name)
 {
 	EQEmuLogSys::CloseFileLogs();
 
+	if (!File::Exists(path.GetLogPath())) {
+		LogInfo("Logs directory not found, creating [{}]", path.GetLogPath());
+		File::Makedir(path.GetLogPath());
+	}
```

Comment:
This directory creation occurs too late, because path manager would have already passed on setting the directory, so `path.GetLogPath()` will return empty string.